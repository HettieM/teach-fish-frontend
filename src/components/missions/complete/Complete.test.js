import React from "react";
import { render, screen } from "@testing-library/react";
import { BrowserRouter } from "react-router-dom";
import renderer from "react-test-renderer";
import Complete from "./Complete";

// test("", () => {
//   render(<Complete />);
//   screen.getByText("NEXT MISSION UNLOCKED");
// });

test("", () => {
  const tree = renderer
    .create(
      <BrowserRouter>
        <Complete />
      </BrowserRouter>,
    )
    .toJSON();
  //   expect(tree).toMatchSnapshot();
  expect(tree).toMatchSnapshot();
});
