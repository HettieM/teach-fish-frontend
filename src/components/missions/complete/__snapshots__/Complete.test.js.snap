// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mission complete page rendering correctly 1`] = `
Array [
  <canvas
    height={768}
    initialY={16}
    style={
      Object {
        "bottom": 0,
        "left": 0,
        "pointerEvents": "none",
        "position": "absolute",
        "right": 0,
        "top": 0,
        "zIndex": 2,
      }
    }
    width={1024}
  />,
  <h2
    className="sc-AxhCb kqvdZj"
  >
    MISSION COMPLETE!!
  </h2>,
  <div
    className="sc-AxjAm eCTsxI"
  >
    <img
      className="sc-AxiKw dKbEvk"
      src="circle.svg"
    />
    <img
      className="sc-AxirZ gSCDgC"
      src="badge.svg"
    />
  </div>,
  <a
    className="makeStyles-links-1"
    href="/my-missions"
    onClick={[Function]}
  >
    NEXT MISSION UNLOCKED
  </a>,
]
`;
